aim.txt

```diff
@@ -0,0 +1,12 @@
+[Section]
+ Name = AIM
+ Version = 5.1
+ License = Abandonware
+ Description = AOL Instant Messenger. Change host DNS to iwarg.ddns.net at port 5190 to restore functionality. Requires an AIM Phoenix account.
+ Size = 2.47 MB
+ Category = 16
+ URLSite = http://iwarg.ddns.net/
+ URLDownload = http://iwarg.ddns.net/aim/files/aim513036.exe

```

Comment:
Please don't use a `ddns.net` address for this.

Reply:
What is the problem with a ddns.net address? Fill me in. This download is coming from another guy's website(virus checked and all, not that a virus on a ReactOS installation will do much at this stage.)

Reply:
ddns is a service that forwards a hostname to a dynamic ip,
which is usually used to connect one's private computer / synology to the internet, and reach it from the outside.
This means that the hosting most likely will not be reliable, and can result in reports that it's down at times.

Reply:
So let me get this straight. You're asking me to change a link which, for every time I've ever used it, has been functional, by closing this merger that fixes another REALLY important issue(I have to close it because I screwed up the implementation of this merger by using master branch without thinking and now there's no way for me to fix this merger with Git Bash cleanly.) I mean, if it's really THAT bad of an issue, I will, but goodness me, this is embarrassing.